bodhi/tests/server/test_models.py

```diff
@@ -710,6 +710,24 @@ def test_mandatory_days_in_testing_status_truthy(self):
         """Test mandatory_days_in_testing() with a value that is truthy."""
         self.assertEqual(self.obj.mandatory_days_in_testing, 42)
 
+    def test_setting_prefix(self):
+        """Assert correct return value from the setting_prefix property."""
+        self.assertEqual(self.obj.setting_prefix, 'f11')
+
+        # Try putting a - into the name of the release, which should get removed
+        self.obj.name = 'f-11'
+
+        self.assertEqual(self.obj.setting_prefix, 'f11')
+
+    @mock.patch.dict(config, {'f11.status': "It's doing just find, thanks for asking"})
+    def test_setting_status_found(self):
+        """Assert correct return value from the setting_status property when config is found."""
+        self.assertEqual(self.obj.setting_status, "It's doing just find, thanks for asking")
```

Comment:
"doing just *find*" ? ;-)
(totally optional and unrelated to this patch ;-) )

Reply:
Fixed!